src/Altinn.Notifications/Configuration/GeneralSettings.cs

```diff
@@ -24,4 +24,14 @@ public class GeneralSettings
     /// Default sender of email notifications
     /// </summary>
     public string DefaultEmailFromAddress { get; set; } = "noreply@altinn.no";
+
+    /// <summary>
+    /// Start hour of the SMS send window
+    /// </summary>
+    public static int SmsSendWindowStartHour { get; set; } = 9;
```

Comment:
Honestly not sure I like these new properties being static. I assume it's still possible to set the value through appsettings and other configuration sources?

Reply:
created a seperate service for schedule to limit dependencies for controller. Config values moved to NotificationOrderConfig. We could consider renaming the config section to NotificationConfig

Reply:
I changed the name.